mmdt/survey/forms.py

```diff
@@ -24,3 +26,8 @@ def __init__(self, *args, **kwargs):
                     choices = [(choice.id, choice.choice_text) for choice in question.choices.all()]
                     self.fields[field_name] = forms.IntegerField(label=question.question_text, widget=forms.NumberInput(attrs={'type': 'range'}), required=False)
     return SurveyForm
+
+class CustomUserCreationForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = ['username', 'email', 'password1', 'password2']
```

Comment:
I found this class in polls/forms.py too. Shall we use in share location?  we could able to extract all user related info that could able to share blog, poll, survey to some where like user/forms.py then we could able to use from all apps for user related stuff.